core/src/plan/join.rs

```diff
@@ -163,6 +163,116 @@ impl<'a> JoinPlanner<'a> {
         (context, join)
     }
 
+    fn subquery_expr(&self, outer_context: Option<Rc<Context<'a>>>, expr: Expr) -> Expr {
```

Comment:
Why is the change here?

Reply:
Just realized that there was wrong merging.. Thank you 👍